.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+### Details:
+ - *item1*
+ - *...*
+
+### Tickets:
+ - *ticket-id*
+
+### Pull Request Readiness Checklist
+
+See details at https://github.com/openvinotoolkit/openvino/wiki/Contribute#making-a-good-pull-request
+
+- [ ] I agree to contribute to the project under Apache 2 License.
```

Comment:
Do we really want to click all this boilerplate checkboxes all the time? 
Can we just have Details and Tickets sections in the template + Contribution guide with all these thing below?

Reply:
If developer need to click all checkboxes, he can remember something that he loose.
We can reduce the number of checkboxes. Do you have any proposals?

Reply:
Yes, my proposal is to keep only Details and Tickets sections.

Reply:
The short checkbox list can allow to think about different things when you create the PR.
I think we can reduce the number of checkboxes but I don't agree that we need to keep only Details and Tickets sections.
First of all because we also have external contributors which want to improve the OV product.

Reply:
There is a contribution guide for any kind of check-lists. I'm pretty sure that everyone will just mark all of these check boxes automatically and we'll have a bunch of useless text in PR description. 
As for external contributors, I've checked tf and pytorch repo that have much more external contributions, and they don't have PR template at all. That means something...